### DISCUSSION TITLE
`command 'pretext-tools.selectPretextCommand' not found`

### DISCUSSION THREAD

**Discussion by StevenClontz** [oscarlevin/pretext-tools#72] [Q&A] [answered]
Any hints on why I might be getting this error when I'm trying to wire up `pretextbook/pretext` with PreTeXt-tools?

**Comment by StevenClontz** [comment]
Hmm at some point after some codespace rebuilds, it seems to work now. Something to keep an eye on.

**Comment by oscarlevin** [comment, marked as answer]
There was a bug in 0.15 that triggered when a user didn't have a standard source/main.ptx file.  That should be fixed in 0.15.2, which was pushed last night.
